src/CAN/can_motor.cpp

```diff
@@ -0,0 +1,38 @@
+#include "can_motor.h"
```

Comment:
Thinking about it a bit more, since the CAN folder is supposed to be isolated from most of the rest of the codebase, it might make more sense to put `can_motor.cpp` and `can_motor.h` in `src/world_interface` with the rest of the motor files.

It's not ideal to have files in `src/CAN` including files from `src/world_interface`, since the latter is supposed to strictly be a client of the former.